package.json

```diff
@@ -133,6 +133,7 @@
     "react-test-renderer": "18.0.0",
     "service-worker-mock": "^2.0.5",
     "ts-jest": "27.1.4",
+    "ts-node": "9.1.1",
```

Comment:
whats this being used for?

Reply:
ah, need to remove that.. missed the cleanup from the unit test debugging